local/gadgets/codecraft/admin_info/lang/ru/index.php

```diff
@@ -1,3 +1,9 @@
 <?
-$MESS["GD_SECURITY_LEVEL"] = "";
+$MESS["GD_CODECRAFT_SECURITY_LEVEL"] = "";
+$MESS['GD_CODECRAFT_ADDRESS']='Адрес сайта:';
```

Comment:
Global array $MESS must contain unique keys for gadget's strings.
Replace
$MESS['GD_CODECRAFT_ADDRESS']= ...
to
$MESS['GD_CODECRAFT_INFO_ADDRESS']=...
and other keys by parity of reasoning.